implementations/rust/ockam/ockam_api/src/nodes/service/in_memory_node.rs

```diff
@@ -58,10 +58,13 @@ impl Drop for InMemoryNode {
         // because in that case they can be restarted
         if !self.persistent {
             executor::block_on(async {
-                self.node_manager
-                    .delete_node()
-                    .await
-                    .unwrap_or_else(|e| panic!("cannot delete the node {}: {e:?}", self.node_name))
+                // We need to recreate the CliState here to make sure that
```

Comment:
Should this be fixed before landing?

Reply:
Oops, this is also a left-over from testing. Thanks!